pg_activity

```diff
@@ -261,6 +261,14 @@ def get_parser() -> ModifiedOptionParser:
         help="Disable App",
         default=False,
     )
+    # --hide-queries-in-logs
+    group.add_option(
+        "--hide-queries-in-logs",
+        dest="hidequeriesinlogs",
```

Comment:
This would be more readable as `hide_queries_in_logs`.

Reply:
I agree, I used this because all the other option names where squashed like this.
Will used your format, then.

Reply:
done